python/_tskitmodule.c

```diff
@@ -5157,6 +5157,30 @@ TableCollection_has_index(TableCollection *self)
     return Py_BuildValue("i", (int) has_index);
 }
 
+static PyObject *
+TableCollection_equivalent(TableCollection *self, PyObject *args, PyObject *kwds)
+{
+    PyObject *ret = NULL;
+    TableCollection *other = NULL;
+    tsk_flags_t options = 0;
+    int cmp_metadata = false;
+    int cmp_provenance = false;
+    static char *kwlist[] = {"compare_metadata", "compare_provenance" , NULL};
```

Comment:
I think you need "other" at the start of kwlist

Reply:
compare `TableCollection_union`

Reply:
Yep, this is needed. Tests for this function pass after adding.

Reply:
yesssss